### PR TITLE
Add PyTorch example

### PR DESCRIPTION
**Pull request by teddykoker**
Excellent work with Tract! I added an example using PyTorch/ONNX that might make it easier for some to get started.

### CODE REVIEW

**Comment by kali** [comment]
Wow, thanks ! That's great :)

File: examples/pytorch-resnet/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "pytorch-resnet"
```
Comment (kali) [inline examples/pytorch-resnet/Cargo.toml:2]: Can we use example- as a prefix here to be consistent with the other test ?

Comment (teddykoker) [reply examples/pytorch-resnet/Cargo.toml:2]: Yes!

**Comment by kali** [comment]
all good ? :)

**Comment by teddykoker** [comment]
Yup! I added standardization as well; this might need to be added to other examples.

**Comment by kali** [comment]
Yep, but some networks have it built-in as the first stage... And well, the whole point of NN should be not to have to worry too much about a bit of color correction :)

**Comment by kali** [comment]
Merged. Thanks a lot.

**Comment by teddykoker** [comment]
Great! Thank you for your work on this, it is good to see other low level deployment options.

**Pull request merged**